browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -4354,7 +4404,46 @@ index 46169f509995747f4df200c3c08674c7735a875a..a3ee4b0e4804480f50cf2d05d528bd11
  #include "Logging.h"
  #include <wtf/CompletionHandler.h>
  
-@@ -257,26 +258,32 @@ void PolicyChecker::checkNewWindowPolicy(NavigationAction&& navigationAction, Re
+@@ -203,10 +204,12 @@ void PolicyChecker::checkNavigationPolicy(ResourceRequest&& request, const Resou
+     auto requestIdentifier = PolicyCheckIdentifier::create();
+     m_delegateIsDecidingNavigationPolicy = true;
+     String suggestedFilename = action.downloadAttribute().isEmpty() ? nullAtom() : action.downloadAttribute();
++    InspectorInstrumentation::willCheckNavigationPolicy(m_frame, request.url());
```

Comment:
What about same page navigation and other scenarios that are handled locally in frame loader, don't we need them too?